src/styles/theme.scss

```
@@ -66,3 +66,70 @@ $breakpointTablet: 1024px;
     @content;
   }
 }
+
+/** Shared MUI Select styles **/
+@mixin muiSelectRoot {
```

Stylelint (17.4.0)

[error] Expected mixin name "muiSelectRoot" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)